WebApi/Controllers/BotBasicController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Application;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using Telegram.Bot;
+using Telegram.Bot.Types;
+
+namespace WebApi.Controllers
+{
+    [Route("api/bot")]
+    [ApiController]
+    public class BotBasicController : ControllerBase
+    {
+        private readonly TelegramBotClient _telegraBotClient;
+        public BotBasicController(TelegramBot telegramBot)
+        {
+            _telegraBotClient = telegramBot.GetBot().Result;
```

Comment:
very bad to call async inside controller and very bad to call Result, it can lead to deadlock

Reply:
changed to correct way